test/terra/noise/test_standard_errors.py

```diff
@@ -139,32 +115,32 @@ def test_pauli_error_1q_unitary_from_pauli(self):
         """Test single-qubit pauli error as unitary qobj from Pauli obj"""
         paulis = [Pauli(s) for s in ['I', 'X', 'Y', 'Z']]
         probs = [0.4, 0.3, 0.2, 0.1]
-        error = pauli_error(zip(paulis, probs), standard_gates=False)
-
-        target_unitaries = [standard_gate_unitary('x'),
-                            standard_gate_unitary('y'),
-                            standard_gate_unitary('z')]
+        target_unitaries = [Pauli("I").to_matrix(),
+                            Pauli("X").to_matrix(),
+                            Pauli("Y").to_matrix(),
+                            Pauli("Z").to_matrix()]
         target_probs = probs.copy()
-        target_identity_count = 0
-        for j in range(len(paulis)):
-            circ, p = error.error_term(j)
-            name = circ[0]['name']
-            self.assertIn(name, ('unitary', 'id'))
-            self.assertEqual(circ[0]['qubits'], [0])
-            self.remove_if_found(p, target_probs)
-            if name == "unitary":
-                self.remove_if_found(circ[0]['params'][0], target_unitaries)
-            else:
-                target_identity_count += 1
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
```

Comment:
We really shouldn't be catching _all_ warnings within a test - warnings-as-errors is deliberately turned on, and we should configure `warnings.filterwarnings` to catch only the few warnings we really deem acceptable (probably only `DeprecationWarnings` emanating from `qiskit.providers.aer.noise.error.errorutils`, I think?).